### ISSUE TITLE
Installing from source without pip

### ISSUE THREAD

**Issue by lpipes** [cogent3/cogent3#1086]
**Is your feature request related to a problem? Please describe.**
Is there a way to install without pip?
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by GavinHuttley** [comment]
Yes, but the best answer depends on why pip is not desirable. Is it because:

1. you want the bleeding edge developer version so you can update it as patches are pushed in?
2. you do not have admin rights on your computer?

**Comment by GavinHuttley** [comment]
What OS are you on?

**Comment by lpipes** [comment]
ubuntu

**Comment by GavinHuttley** [comment]
Sweet!

A virtual environment is the way to go. [Here's a primer on those](https://realpython.com/python-virtual-environments-a-primer/), but the following "should work". (I'll be away from my computer after this, but will check on progress tomorrow.)

In a location that's useful for you, and making sure you're using a suitable python version (e.g 3.10) do

```
$ python3.10 -m venv c3venv # the c3venv name is arbitrary, call it whatever you like
$ source ~/c3venv/bin/activate # you can make this command an alias
(c3venv) $ pip install cogent3
```

Because this is done in your account, you won't run into permission issues.

It does mean to use cogent3 you will need to make sure the vritual environment is active.
